metric_providers/lm_sensors/abstract_provider.py

```diff
@@ -3,15 +3,19 @@
 # Discussion is here https://github.com/green-coding-berlin/green-metrics-tool/issues/39
 
 import os
-
+import sys
 #pylint: disable=import-error
 from global_config import GlobalConfig
+current_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(f"{current_dir}/../../lib")
+import utils
 from metric_providers.base import BaseMetricProvider
 
 class LmSensorsProvider(BaseMetricProvider):
 
     def _create_options(self):
-        provider_config = GlobalConfig().config['measurement']['metric-providers'][self._provider_config_path]
+        provider_config = GlobalConfig().config['measurement']['metric-providers'][utils.get_architecture()]\
```

Comment:
I changed this in the PR to have the static key "linux". I think it makes no sense to have this provider architecture agnostic, when it is only available on linux anyway (see: https://formulae.brew.sh/formula/lm-sensors)

Reply:
changed to ['linux']